src/partials/navbar-end.hbs

```diff
@@ -110,46 +57,46 @@
-                <li><a href="/docs/nvl/current/" class="project-link">Neo4j Visualization Library</a></li>
+                <li><a href="{{#with (docs-driver-manual-url page 'python-manual')}}{{{this}}}{{/with}}" class="project-link">Python</a></li>
+                <li><a href="{{#with (docs-driver-manual-url page 'go-manual')}}{{{this}}}{{/with}}" class="project-link">Go</a></li>
+                <li><a href="{{#with (docs-driver-manual-url page 'java-manual')}}{{{this}}}{{/with}}" class="project-link">Java</a></li>
+                <li><a href="{{#with (docs-driver-manual-url page 'jdbc-manual')}}{{{this}}}{{/with}}" class="project-link">JDBC</a></li>
+                <li><a href="{{#with (docs-driver-manual-url page 'javascript-manual')}}{{{this}}}{{/with}}" class="project-link">JavaScript</a></li>
+                <li><a href="{{#with (docs-driver-manual-url page 'dotnet-manual')}}{{{this}}}{{/with}}" class="project-link">.Net</a></li>
+                <li><a href="/docs/graphql-manual/current/" class="project-link">GraphQL</a></li>
+                <li><a href="/docs/nvl/current/" class="project-link">Visualization Library</a></li>
                 <li><a href="/docs/ogm-manual/current/" class="project-link">OGM Library</a></li>
-                <li><a href="https://docs.spring.io/spring-data/neo4j/docs/current/reference/html/#reference" class="project-link">Spring Data Neo4j</a></li>
-                <li><a href="/docs/http-api/current/" class="project-link">HTTP API</a></li>
-                <li><a href="/docs/query-api/current/" class="project-link">Neo4j Query API</a></li>
+                <li><a href="https://docs.spring.io/spring-data/neo4j/docs/current/reference/html/" class="project-link">Spring Data Neo4j</a></li>
+                <li><a href="/docs/query-api/current/" class="project-link">Query (HTTP) API</a></li>
                 <li><a href="/docs/bolt/current/" class="project-link">Bolt</a></li>
               </ul>
             </div>
 
             <div class="navbar-item project">
               <span class="project-name" href="/docs/connectors">Connect data sources</span>
               <ul class="project-links">
-                <li><a href="/docs/spark/current/" class="project-link">Neo4j Connector for Apache Spark</a></li>
-                <li><a href="/docs/kafka/" class="project-link">Neo4j Connector for Apache Kafka</a></li>
-                <li><a href="/docs/cdc/" class="project-link">Change Data Capture (CDC)</a></li>
-                <li><a href="/docs/dataflow-bigquery/" class="project-link">BigQuery to Neo4j</a></li>
-                <li><a href="/docs/dataflow-google-cloud/" class="project-link">Google Cloud to Neo4j</a></li>
+                <li><a href="/docs/cdc/current/" class="project-link">Change Data Capture (CDC)</a></li>
+                <li><a href="/docs/spark/current/" class="project-link">Apache Spark</a></li>
```

Comment:
General comment for all the connectors. I'm not sure if we can use _just_ the name of the platform without having _Neo4j Connector_ in the title. It becomes a potential copyright issue.